resources/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: objectstorage-provisioner
+  labels:
+    app.kubernetes.io/part-of: container-object-storage-interface
+    app.kubernetes.io/component: driver-ceph
+    app.kubernetes.io/version: main
+    app.kubernetes.io/name: cosi-driver-ceph
+type: Opaque
+#stringData:
+#  DRIVER_PREFIX: ""
```

Comment:
Is it really necessary to read this information from a configmap? It seems like one extra step that isn't really necessary if someone can instead change an env var's value on the deployment spec.

Alternately, would a CLI flag be a better way of accepting different user inputs to the COSI drive? Ceph-CSI uses flags and not env vars, which seems better to me. Flags can be validated more easily than env vars, and they have better documentation defined internally to the application.

Reply:
Done